src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -174,4 +177,19 @@ private function isMaxRangeSlider(string $name): bool
     {
         return mb_substr($name, 0, mb_strlen(self::MAX_PREFIX)) === self::MAX_PREFIX;
     }
+
+    private function isPropertyFilter(string $filterName, string $filterValue): bool
+    {
+        return mb_strpos($filterValue, sprintf('%s-', $filterName)) === 0;
+    }
+
+    private function handlePropertyFilter(
+        string $filterName,
+        string $filterValue,
+        SearchNavigationRequest $searchNavigationRequest
+    ): void {
+        $parsedFilterValue = explode('-', $filterValue);
```

Comment:
* What happens if my filter value is `some-value`? This would cause the query parameter to be `filtername-some-value`.
* What happens if my filter name is `some-filter`? This would cause the query parameter to be `some-filter-value`.

Add tests that cover that this does not cause any issues.